runtime/Go/antlr/atn_config_set.go

```diff
@@ -6,12 +6,13 @@ package antlr
 
 import "fmt"
 
+// ATNConfigSet extends ATNConfig in the form of an unordered set.
 type ATNConfigSet interface {
 	hash() int
-	Add(ATNConfig, *DoubleDict) bool
+	Add(ATNConfig, *doubleDict) bool
 	AddAll([]ATNConfig) bool
 
-	GetStates() *Set
+	getStates() *_set
```

Comment:
Leading underscores aren't conventional. If I recall correctly, golint would flag this. It would be better to just name it "set".